tensorflow/compiler/xla/stream_executor/rocm/rocblas_wrapper.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 #ifndef TENSORFLOW_COMPILER_XLA_STREAM_EXECUTOR_ROCM_ROCBLAS_WRAPPER_H_
 #define TENSORFLOW_COMPILER_XLA_STREAM_EXECUTOR_ROCM_ROCBLAS_WRAPPER_H_
 
-#include "rocm/include/rocblas.h"
+#include "rocm/include/rocblas/rocblas.h"
```

Comment:
Does this depend on the ROCM version which path is valid?
If yes, I think you need some #ifdef here.

Reply:
I'm not sure if this is dependent on the ROCM version.

Reply:
What happens is that this fails in our ROCM Kokoro. It still uses ROCM 5.0, therefore I thought maybe the path depends on the ROCM version?
Unfortunately we cannot merge a PR if it breaks our testing.

Reply:
We have moved our Kokoro to ROCM 5.3, and now the change worked. Which confirms the include path is version specific. But I guess we don't care to support ROCM 5.0 anymore?